.github/workflows/ansible-test.yml

```diff
@@ -38,9 +40,15 @@ jobs:
           ${{ matrix.name }}/build.sh
         working-directory: ./ansible-test/
 
+      # Right now it is not possible to use some of the images (Debian Bullseye, ArchLinux) with podman in GitHub Actions,
+      # while others (CentOS 8 Stream) work fine. This is not specific to the images in this repository, the official
+      # ansible-test Ubuntu 20.04 image also does not work with podman in GHA, but does work with Docker in GHA.
```

Comment:
I created an issue so we don't forget about it: https://github.com/ansible-community/images/issues/9